### ISSUE TITLE
[Proposal] Revise and Increase the Withdrawal Fees for SFL to Sustain the Health of the Project 

### ISSUE THREAD

**Issue by EdBoyette** [sunflower-land/sunflower-land#984]
**Proposal add additional fee brackets to reduce the linear decay of SFL Token price to sustain the project, and to provide adequate funding for long term development.**  

Proposed new withdrawal fee brackets are:
A. 50% fee withdrawals less than 10 SFL
B. 40% fee for withdrawals less than 20 SFL
C. 30% fee for withdrawals less than 30 SFL

Withdrawal fees were established to according to the documents to: 

> 
> - "Finance the project, 
> - Stabilize the economy
> - Reward holders 
> - Disincentivize click farmers
> - and Support the Wishing Well".

From inspection of Polygonscan and Dexscreener ~75%+ of current withdrawal and subsequent sells are less than 10 SFL.

Measuring the price decay there is a clear linear degradation (a linear function can be approximated) of SFL price from the constant daily sell-off of SFL.  At the current rate of decline SFL will reach 5 SFL per 1 Matic by the end of May.

This level of sell-off and price decay WILL NOT sustain the long-term development and maintenance of the project.  This level of price decline disincentivizes providers of liquidity and users who hope to see the project to thrive over a period of years.

Increasing the withdrawal fee will curtail daily sell off and encourage investing in the game and a more sustainable perspective. Players who save for longer would still be able to withdraw at less punitive rates, with longer periods between withdrawals. 

This compromise will encourage a usage style that promotes the sustainability of the project, protects liquidity providers, and prevent a "tragedy of the commons".

Thank you for your consideration and suggestions.

**Comment by Aarval** [comment]
I like the idea, feel like a +20% increase across the board is a little steep but I've not run the numbers against sell off. Another suggestion could be increase the withdrawal fee by 10% but keep wishing rewards the same as well as offer additional rewards like seed/resource airdrops to those providing LP in the wishing well. 

This would still provide incentive to those providing LP and would keep the WW pot healthy. Having the WW rewards dynamically change based on how much SFL is in rewards pot. The less SFL in rewards  the more seeds/resources you receive as a reward instead of SFL

**Comment by Rainmakerzz** [comment]
Most definitely no.

I always find it extremely repulsive when a project try to gate my funds in any way. In effect instituting capital controls, which in the history of mankind failed 100% of the time. It will fail here as well, if instituted. You dont increase the support of a project if you force the players to withdraw less frequently, on the contrary, it makes people abandon the game, and not put into any money in the first place.
You want SFL price to sustained? Well if there would be more stuff to spend your SFL, people wouldnt sell it on the market. Keep adding new features and inspire people to want to play the game, and the token will appreciate.
Taxing the average guy to the death is the surest way to ruin a game. A lot of people wont even onboard if you tell them you have up to 50% tax rate on withdrawals. 
I pay enough taxes in RL, I dont need a game where they skin me alive.

For the record I didnt withdraw a single SFL since I started the game early April.

**Comment by RSX-0806** [comment]
> A. 50% fee withdrawals less than 10 SFL
Agree with this

**Comment by RSX-0806** [comment]
A. 50% fee withdrawals less than 10 SFL.
Agree with this

**Comment by Yougottabekiddingme123** [comment]
I agree with the proposal in general. I think something needs to be done in order to prevent the inevitable slide of sfl. 

Having checked the data on Polygonscan and Dexscreener it is indeed obvious that the continous sell pressure from small withdrawals is forcing the price down. Increasing the tax on tiny withdrawals will encourage players to wait a little longer before withdrawing and thus relieve some sell pressure.

Alternative options include keeping the tax rate as it is but putting a limit on the frequency of withdrawals.

The suggested changes above are not as drastic as it may seem because players will not withdraw at 50%. They will collect and withdraw at a lower rate. In otherworld, it is not the same as capital controls because it does not limit the amount that you can withdraw, it only encourages them to delay their withdrawal.

**Comment by SuperSynapse** [comment]
So people will just wait til they get more SFL before they sell.

I don't understand why everyone wants to increase withdrawal fees. **If you're bullish on the game, buy the dip**. Invest the SFL, and make gains.

Realize as well that a good part of the **SFL LP going down in value, is that Matic is going down in value** as well. This will also be **the inverse if Matic starts to go up in value again in the future, so will SFL in the pool due to it's pairing**. If stability is more desirable, then SFL should be paired with USDC, but it will also dampen the uptrend when Matic and the bull market resumes.

**Realize by creating an LP you are agreeing to be both entry liquidity for people buying SFL, as well as exit liquidity to people looking to sell.**

-------------
I believe players playing within the rules should be allowed to play the game however they want, be it investing in NFTs, stacking resources/animals, or even farming to sell, obviously with incentives being to scale farms vertically with NFT stacking.

If you want to relieve the selling pressure, **the real focus needs to be multi-accounting.** I've said this many times. I am aware of many people with 50+ farms and none of them are getting banned, and this is just a small piece of the multi-farming going on. The issue isn't SFL withdrawal fees, or forcing people to buy the NFTs, t**he issue is multi-accounting and people farming as a "job" devaluing everyone else.** 

I'd estimate that the majority of farms are owned by people multi-accounting. If there is a rule of 1 farm per person, **it needs to be enforceable; otherwise they are the winners and people playing within the rules are the losers.**

------------

This is going to be a bold statement, and I mean it in the most polite way possible. The real **conflict of interest** here is that if the problem is truly addressed, the devs will see **significantly less revenues** as **real players create less farms for 10 Matic, sync their farms less, sell their SFL less, and sell their NFTs less**.

It will be a short term pain, especially in a bear market to stop, but long term this is one of the single best things they can do for the future of the game and community.

**Comment by SuperSynapse** [comment]
> From inspection of Polygonscan and Dexscreener ~75%+ of current withdrawal and subsequent sells are less than 10 SFL.

Much of this is trading bots that arbitrage between pools and have nothing to do with the game.

For instance if there are 3 SFL pools:
- SFL/Matic
- SFL/USDC
- SFL/USDT

If SFL is worth $0.20 in all the pools, but then someone buys from the SFL/Matic pool, in that pool SFL is worth more. So the bots will buy a small amount from the other two pools, and sell into the SFL Matic pool for a small profit (usually a few cents)

This is also true with the other coins, for instance if someone buys a large amount of Matic from a Matic/USDC pool, now Matic is worth more in that pool, so a bot will buy SFL from the SFL/USDC pool, swap it for Matic in the Matic/SFL pool, and then sell it into the original Matic/USDC pool the large buy originated from.

This is normal activity. If you'd like to learn more about this, look up "arbitrage" and "Liquidity pools". Very normal activity.

**Comment by brian-man** [comment]
> I agree with the proposal in general. I think something needs to be done in order to prevent the inevitable slide of sfl.

If it's inevitable, then nothing you change will matter.

**Comment by EdBoyette** [comment]
> @EdBoyette
> 
> > From inspection of Polygonscan and Dexscreener ~75%+ of current withdrawal and subsequent sells are less than 10 SFL.
> 
> Much of this is trading bots that arbitrage between pools and have nothing to do with the game.
> 
> For instance if there are 3 SFL pools:
> 
> * SFL/Matic
> * SFL/USDC
> * SFL/USDT
> 
> If SFL is worth $0.20 in all the pools, but then someone buys from the SFL/Matic pool, in that pool SFL is worth more. So the bots will buy a small amount from the other two pools, and sell into the SFL Matic pool for a small profit (usually a few cents)
> 
> This is also true with the other coins, for instance if someone buys a large amount of Matic from a Matic/USDC pool, now Matic is worth more in that pool, so a bot will buy SFL from the SFL/USDC pool, swap it for Matic in the Matic/SFL pool, and then sell it into the original Matic/USDC pool the large buy originated from.
> 
> This is normal activity. If you'd like to learn more about this, look up "Arbitrage" and "Liquidity Pools".

Thank for mentioning this factor - I'm familiar with these mechanisms the volume in these alternative pools doesn't suggest it's a significant portion of the linear decline.

The SFL/USDC pool has approximately 20% of the 24 hour volume of SFL/WMATIC and is more balanced.

**Comment by adamhannigan** [comment]
Thanks for the contribution and feedback! ❤️ 

 We are moving towards using an ideas portal to help capture and vote on community ideas. Github will only be used for bug/technical reports. We are only a small team and it is becoming difficult for us to respond individually to every case so we have began using community voting software.   

 If you could please upload your idea to the following community portal it would be greatly appreciated 🤗   

 https://sunflower-land.ideas.aha.io/

 You can vote on similar ideas and help bring ideas to our attention. Thanks again for being apart of this community!
